components/layout/legacy_variables.styl

```diff
@@ -0,0 +1,53 @@
+// TODO: Delete these as they are eliminated
+// colors
+closed = #4B3D67 // #616161
+public = #17AC10
+private = #B60202
+textgray = #585858 // text, titles
+lightgrayplus = rgb(203, 203, 203) // 1px borders
+lightgray = #9D9D9D // borders, captions, timestamp
+backgroundgray = #F7F7F7 // search bar
+secondbackgroundgray = #F0F0F0 // search results, dropdown
+bordergray = #E7E7E5 // borders
+hovergray = #050505 // any hovered element
+success = #00C5FF
+premium = #00C5FF
+alert = #E24D4D
+modal_bg = rgba(58, 58, 58, 0.85)
+lighthighlight = #FDFFDB
+highlight = #EFFF00
+special = #E7DB5F
+
+// TODO: Extract into UI-scoped z-index file
+// z-index
+header_z = 4001
+dropdown_z = 4002
+loader_z = 5001
+modal_z = 6001
+lightbox_z = 6001
+rotate_z = 7001
+
+// TODO: Extract in UI-scoped font file
+// fonts
+font_sans = 'Arial', 'Helvetica Neue', 'Helvetica', sans-serif
+font_serif = 'Times New Roman', serif
+font_mono = Menlo, Monaco, 'Bitstream Vera Sans Mono', Consolas, Courier, monospace
+
+// TODO: Delete these as they are eliminated
+// font sizes
+s_tiny = 10px
+s_caption = 12px
+s_small = 13px
+s_normal = 14px
+s_form = 16px
+s_middle = 18px
+s_feed = 23px
+
+// TODO: Delete these as they are eliminated
+// units
+unit = 20px
+s_margin = (unit / 2)
+block_width = 315px
+block_width_mar = block_width + (s_margin * 2)
+topbar_h = 50px
+container_pad = 20px
```

Comment:
There's not a huge amount of use for these. The pattern I've been using is UI components have their own variable files which then get required at the top of everything. So you get local isolation but can use them anywhere.